api/scripts/legal-documents/convert-users-orga-cgu-data.js

```diff
@@ -0,0 +1,92 @@
+import 'dotenv/config';
+
+import { DomainTransaction } from '../../lib/infrastructure/DomainTransaction.js';
+import { LegalDocument } from '../../src/legal-documents/domain/models/LegalDocument.js';
+import * as legalDocumentRepository from '../../src/legal-documents/infrastructure/repositories/legal-document.repository.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+const { TOS } = LegalDocument.TYPES;
+const { PIX_ORGA } = LegalDocument.SERVICES;
+
+const BATCH_SIZE = 1000;
+const THROTTLE_DELAY = 100;
+
+export class ConvertUsersOrgaCguData extends Script {
+  constructor() {
+    super({
+      description: 'Convert users orga CGU data to legal-document-versions-user-acceptances',
+      permanent: false,
+      options: {},
+    });
+  }
+
+  async handle({ logger }) {
+    const tosOrgaLegalDocument = await legalDocumentRepository.findLastVersionByTypeAndService({
+      type: TOS,
+      service: PIX_ORGA,
+    });
+
+    if (!tosOrgaLegalDocument) {
+      logger.info(`No legal document found for type:${TOS}, service:${PIX_ORGA}`);
+      return;
+    }
+
+    const knexConnection = DomainTransaction.getConnection();
+
+    let hasMoreUsers = true;
+    let offset = 0;
+
+    while (hasMoreUsers) {
+      const batch = await this.#findOrgaCguAcceptedUsers({ knexConnection, offset });
```

Comment:
Ajouter une log pour le voir l'avancement
```suggestion
    while (hasMoreUsers) {
      logger.info(`Batch #${offset / BATCH_SIZE + 1}`);
      const batch = await this.#findOrgaCguAcceptedUsers({ knexConnection, offset });
```